.github/workflows/pull-request.yml

```diff
@@ -41,28 +46,47 @@ jobs:
           msg: |
             ## :white_check_mark: No Breaking Changes
           check_for_duplicate_msg: false
-  protoc:
+
+  java:
     runs-on: ubuntu-latest
     steps:
       - uses: actions/checkout@v2
-      - name: Setup Java
-        uses: actions/setup-java@v1
+
+      - uses: actions/setup-java@v1
         with:
           java-version: 11
-      - name: Setup Node
-        uses: actions/setup-node@v1
+          java-package: jdk
+
+      - name: setup protoc
+        run: ./install_proto.sh
+
+      - name: make
+        run: make PROTOC=./bin/protoc LANGUAGE=java OUTPUT=./java/src/main/java
```

Comment:
Warum ist es sinnvoll das outdir konfigurierbar zu machen?
Der Aufruf erscheint mir sehr kompliziert im vergleich zu vorher wo ich im dev Modus nur schnell “make all” eingegeben habe.

Reply:
to be language independent